lib/rubocop/cop/style/frozen_string_literal_comment.rb

```diff
@@ -12,7 +12,7 @@ class FrozenStringLiteralComment < Cop
         include ConfigurableEnforcedStyle
         include FrozenStringLiteral
 
-        MSG = 'Missing frozen string literal comment.'.freeze
+        MSG = 'File must begin with "# frozen_string_literal: true".'.freeze
```

Comment:
Ok. How about something like:

```ruby
MSG = 'Comment must be present near top of file: `# frozen_string_literal: true`.'
```

It's a difficult thing to phrase clearly without getting too wordy :/